src/main/java/com/chargerrobotics/utils/SmartDashboardHelper.java

```diff
@@ -0,0 +1,12 @@
+package com.chargerrobotics.utils;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+
+public class SmartDashboardHelper {
+
+  public double getNumber(String key, double defaultValue) {
+    return SmartDashboard.getNumber(key, defaultValue);
```

Comment:
What's your reasoning for adding this method? It seems as if it would be just as easy to use `SmartDashboard.getNumber(key, default)` wherever you would call this method.

Reply:
It had allowed us to mock smartDashboard, but is unneeded now. Will be deleted in the new push.